src/Entity/Entity.php

```diff
@@ -16,21 +16,25 @@ public function __construct(array $data = [])
         $this->data = $data;
     }
 
+    #[\ReturnTypeWillChange]
```

Comment:
that attribute presented in php8 https://www.php.net/manual/en/class.returntypewillchange.php

our lib states it would work on anything more than 5.5

meaning, it would also work on 5.5. but it wont

Reply:
Yes, exactly, as function return type not added in php5.5, we've used temporary 
```php
#[\ReturnTypeWillChange]
```
function's dynamic return type to fix that return type issue in PHP8.x. So, it wouldn't create an issue in PHP5.5, as for PHP5.5 it's just a comment out code.

<img width="799" alt="Screen Shot 2023-04-27 at 1 35 33 PM" src="https://user-images.githubusercontent.com/17502625/234792514-972057d6-1bc8-49f7-94c6-8c840eb853d5.png">